docusaurus/docs/reference/tunnelers/linux/linux-tunnel-troubleshooting.md

```diff
@@ -23,29 +23,69 @@ journalctl -xeu ziti-edge-tunnel.service
 
 ## Intercepting or hosting not working
 
-You may inspect the loaded identity and router info for a running `ziti-edge-tunnel` by dumping it to stdout or the systemd journal with an IPC command, or you may signal to dump the identities' info to a file.
+You may inspect the loaded identity and router info for a running `ziti-edge-tunnel` by dumping it to stdout or the systemd journal with an IPC command. Alternatively, you may signal to dump the identities' info into a file.
+
+```bash
+# dump identities info to std our journal if systemd unit with IPC command
+./ziti-edge-tunnel dump
+```
+
+```bash
+# dump identities info to a file and inspect
+sudo pkill -USR1 -f ziti-edge-tunnel
+less /tmp/ziti-dump.964315.dump
+```
 
-  ```bash
-  # dump identities info to std our journal if systemd unit with IPC command
-  ./ziti-edge-tunnel dump
-  ```
+Inspect the full command-line of the tunneler process
+
+```bash
+$ pgrep -f ziti-edge-tunnel | xargs ps -fww
+UID          PID    PPID  C STIME TTY      STAT   TIME CMD
+ziti      872509       1  0 10:42 ?        Ssl    1:54 /opt/openziti/bin/ziti-edge-tunnel run --verbose= --dns-ip-range=100.64.0.1/12 --identity-dir=/opt/openziti/etc/identities
+```
+
+Confirm the tunneler's tun interface is UP
```

Comment:
is this really necessary? feels like we should just verify ziti-edge-tunnel is running and that's enough?